packages/@webex/plugin-meetings/src/member/util.ts

```diff
@@ -226,21 +218,28 @@ MemberUtil.isLiveAnnotationSupported = (participant) => {
 
 /**
  * utility method for audio/video muted status
- * @param {String} status
- * @param {String} accessor
- * @returns {Boolean}
+ * @param {any} participant
+ * @param {String} statusAccessor
+ * @param {String} controlsAccessor
+ * @returns {Boolean | undefined}
  */
-MemberUtil.isMuted = (status: string, accessor: string) => {
-  if (status) {
-    if (status[accessor] === _RECEIVE_ONLY_) {
-      return true;
-    }
-    if (status[accessor] === _SEND_RECEIVE_) {
-      return false;
-    }
+MemberUtil.isMuted = (participant: any, statusAccessor: string, controlsAccessor: string) => {
+  // check remote mute
+  const remoteMute = participant?.controls?.[controlsAccessor]?.muted;
+  if (remoteMute === true) {
+    return true;
   }
 
-  return null;
+  // check local mute
+  const localStatus = participant?.status?.[statusAccessor];
+  if (localStatus === _RECEIVE_ONLY_) {
+    return true;
+  }
+  if (localStatus === _SEND_RECEIVE_) {
```

Comment:
not strictly related to this PR, but I wonder why the existing code is only checking for _RECEIVE_ONLY_ and _SEND_RECEIVE_, why is it ignoring the other 2 possible values: _SEND_ONLY_ and _INACTIVE_, @arun3528 do you know?

Reply:
recently someone was looking into it , @meta-dipanshu-sharma  was it you ?